sdk/eventhubs/azure-messaging-eventhubs/src/main/java/com/azure/messaging/eventhubs/PartitionPumpManager.java

```diff
@@ -376,12 +374,12 @@ private void endProcessTracingSpan(Context processSpanContext, Signal<Void> sign
         }
 
         Object spanObject = spanScope.get();
-        if (spanObject instanceof Closeable) {
-            Closeable close = (Closeable) spanObject;
+        if (spanObject instanceof AutoCloseable) {
```

Comment:
The error message in [messages](https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/eventhubs/azure-messaging-eventhubs/src/main/resources/eventhubs-messages.properties#L1) file should be updated as well.